eth2/types/src/newtypes.rs

```diff
@@ -0,0 +1,88 @@
+// Many patterns copied from the slot_epoch.rs file in the
+// https://github.com/sigp/lighthouse repo with some small modifications.  Specifically, added new
+// types, and removed code not needed for this repo.
+// The analog of the `Root` struct in this repo is H256 in lighthouse (and in some other
+// ethereum-focused crates). Decided to use `Root` in this repo instead of `H256` because the spec
+// refers to the data type as a `Root`.  This can be changed in the future if necessary.
+use fixed_hash::construct_fixed_hash;
+
+// Necessary for impl_common macro
+use serde::{Deserialize, Serialize};
+use slog;
```

Comment:
We will install our own logging system, right? Where is this used?

Reply:
Let me look, possible I can remove this.

Reply:
Yeah - you don't need this. We'll insert our own logging - which I encourage you to do. But you mayh want to use slog. Up to you!

Reply:
Good call, wasn't needed, removed it.